rtt_roscomm/include/rtt_roscomm/passthrough_callback_queue.hpp

```diff
@@ -0,0 +1,232 @@
+
```

Comment:
nit: remove empty line (and add a license header instead)

Reply:
Not adding a license header, otherwise is more problematic to maintain. Instead, the file inherits the `LICENSE` in the root of the package (or project).